sdk/core/core-client/review/core-client.api.md

```diff
@@ -324,6 +337,9 @@ export interface ServiceClientOptions {
     baseUri?: string;
     credential?: TokenCredential;
     httpsClient?: HttpsClient;
+    parseXML?: (str: string, opts?: {
+        includeRoot?: boolean;
+    }) => Promise<any>;
```

Comment:
would it make sense to also pass `stringifyXML` to serializer now, or wait until it is needed?

Reply:
nvm it's down there.